examples/resources/materialize_connection_postgres/import.sh

```diff
@@ -1,2 +1,6 @@
-#Connections can be imported using the connection id:
+# Connections can be imported using the `connection id`:
 terraform import materialize_connection_postgres.example <connection_id>
+
+# You can find the `connection_id` from the following query:
```

Comment:
Can we leave this out of the example for now? If we are going to include these queries to lookup the resource id  for imports, we should probably include them for all examples in the directory

Reply:
Yup can do 👍 . Was just a bit confusing initially as I was trying the "name" thinking it was the identifier.